### PR TITLE
Consistent keybinding dashboard/mu4e

### PR DESCRIPTION
**Pull request by Airwan**
This is a first attempt to install mu4ed minor mode into mu4e (header) views so that keybindings between dashboard/mu4e  become consistent. My hope is to avoid completely bookmarks configurations in mu4e to control keybindings from dashboard only.

I hope we can discuss on that point and look forward some feedback. Thanks   

(first PR of my life :o)

### CODE REVIEW

**Comment by Airwan** [comment]
Note : mu4e-index-updated-hook not works very well with mu4e 1.6. This works with 1.7 (see https://github.com/djcb/mu/pull/2111#issuecomment-950115339).

**Comment by Airwan** [comment]
Hello. Maybe it is the good way indeed. However, I like the simplicity offered with your solution (keymap inside the dashboard config). Modifying a minor mode at running time seems to be not very easy (up to my knowledge) while reloading the local keymap sems to be toggling the mode off and then on again.  I don't know really in fact. 

Also an other key point for me, is to have something like @cmacrae (https://github.com/rougier/mu4e-dashboard/pull/18#issue-794636941) proposed. I am dealing with a similar version toggling the side bar show/hide and implementing a "context". I mean an option that change the dashboard file in use as well as the mu4e context. So keeping keymap inside the dashboard config file seems to be a better solution to pass from one context to an other with its own keys, is'nt it?

File: mu4e-dashboard.el [hunk 1]
```diff
@@ -45,6 +45,10 @@
   "Provides a new Org mode link type for mu4e queries."
   :group 'comm)
 
+(defcustom mu4e-dashboard-file "~/.emacs.d/my-mu4e-dashboard.org"
```
Comment (rougier) [inline mu4e-dashboard.el:48]: I would prefer `~/.emacs.d/mu4e-dashboard.org`

**Comment by rougier** [comment]
Sounds good. I've only a minor comment (see above) and we can merge.

**Comment by Airwan** [comment]
It's down! Should I do something now so that you can merge?

**Comment by rougier** [comment]
Sorry, Im' very late. Thanks gain for the patch.

**Pull request merged**